hs_core/models.py

```diff
@@ -610,10 +610,13 @@ def update(cls, element_id, **kwargs):
                 except TypeError:
                     raise TypeError("Not a valid date value.")
                 if dt.type == 'created':
-                    raise ValidationError("Resource creation date can't be changed")
+                    #raise ValidationError("Resource creation date can't be changed")
+                    dt.start_date = start_dt
+                    dt.save(force_update=True)
                 elif dt.type == 'modified':
-                    dt.start_date = now().isoformat()
-                    dt.save()
+                    # dt.start_date = now().isoformat()
+                    dt.start_date = start_dt
+                    dt.save(force_update=True)
```

Comment:
@hyi @ironfroggy These saves appear to work, at least for a little bit.  Subsequent queries within my debug session see the date as being updated, but the update does not stick so that dates remain the same in the web interface.  I am wondering if there is some cacheing going on, or lower-level validation that is reversing the change?